_i18n/en.yml

```diff
@@ -141,14 +140,42 @@ sponsorships:
+  community_start: The best way to start contributing to this workgroup is to join the chatrooms and introduce yourself, stating your interests and the skills you would like to offer to the community.
+  devworkgroup: Development Workgroup
+  devworkgroup_descr: "This workgroup reunites Monero developers working on the wallets and daemon maintained by the Core Team. This workgorup consists of two main parts: the CLI and the GUI teams. These teams coordinate using two different chatrooms."
+  devworkgroup_start: It's a good idea to take a look at open issues on both repositories to have an idea of what could be fixed or improved. Both codebases, but especially the core repository, can be quite challenging, so it's suggested to first write in the chatroom introducing yourself, your skills and what you wish to contribute to.
+  chatgeneral: "Chat (general):"
+  chatgui: "Chat (GUI):"
+  webworkgroup: Website Workgroup
+  web_descr: Another dev-focused workgroup where to coordinate the development of the website. Most devs in there are part of the development workgroup. The website workgroup was created mostly to have discussions related to the website all in one place.
+  web_start: It's always a good idea to join the chatroom and talk directly with the members of the workgroup, but if you prefer to get started by yourself, take a look at the readme in the GitHub repository. It contains step by step instructions to edit every single part of this website.
+  localization: Localization Workgroup
+  localization_descr: One of the oldest Monero Workgroups. It's formed by contributors working on translating Monero-related software and documentation. It's strongly suggested to contact the workgroup before working on translations.
+  localization_start: Contact the workgroup on the chatroom or open an issue on the GitHub repository. You can also start translating immediately using the Weblate platform. Just make sure to read the guide for translators first.
+  locplatform: "Localization platform:"
+  outreach_descr: This workgroup is focused on getting Monero out there, gathering artists, writers and anyone who wish to make Monero known to the World.
+  outreach_start: The members of the Outreach workgroup heavily use Telegram for coordination and matrix/freenode in minor part. If you want to help, you have to contact the workgroup on one of this channels. They are quite structured, so you'll probably be directed to a coordinator.
+  telegram: "Telegram chat:"
+  mrl_descr: As the name suggests, the Monero Research Lab is the workgroup focused on Monero related research. It's formed by scientists trying to find a way to improve Monero's protocol and features. Most of the protocol improvements who make Monero amazing, like CLSAG and @Bulletproofs, are the result of the work of this workgroup.
```

Comment:
I know this PR has already been merged, but this should have been:
s/find a way/find ways/   <-- are we really only looking for one way to improve? nope.
s/improvements who make/improvements that make/   <-- the improvements are things, not people.

Reply:
#1378